app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -84,44 +84,64 @@ object ProjectDao
     val slug = Project.slugify(newProject.name)
-    (fr"INSERT INTO" ++ tableF ++ fr"""
-        (id, created_at, modified_at, created_by,
-        modified_by, owner, name, slug_label, description,
-        visibility, tile_visibility, is_aoi_project,
-        aoi_cadence_millis, aois_last_checked, tags, extent,
-        manual_order, is_single_band, single_band_options, default_annotation_group,
-        extras)
-      VALUES
-        ($id, $now, $now, ${user.id},
-        ${user.id}, $ownerId, ${newProject.name}, $slug, ${newProject.description},
-        ${newProject.visibility}, ${newProject.tileVisibility}, ${newProject.isAOIProject},
-        ${newProject.aoiCadenceMillis}, $now, ${newProject.tags}, null,
-        TRUE, ${newProject.isSingleBand}, ${newProject.singleBandOptions}, null,
-        ${newProject.extras}
+    for {
+      project <- (fr"INSERT INTO" ++ tableF ++ fr"""
+          (id, created_at, modified_at, created_by,
+          modified_by, owner, name, slug_label, description,
+          visibility, tile_visibility, is_aoi_project,
+          aoi_cadence_millis, aois_last_checked, tags, extent,
+          manual_order, is_single_band, single_band_options, default_annotation_group,
+          extras)
+        VALUES
+          ($id, $now, $now, ${user.id},
+          ${user.id}, $ownerId, ${newProject.name}, $slug, ${newProject.description},
+          ${newProject.visibility}, ${newProject.tileVisibility}, ${newProject.isAOIProject},
+          ${newProject.aoiCadenceMillis}, $now, ${newProject.tags}, null,
+          TRUE, ${newProject.isSingleBand}, ${newProject.singleBandOptions}, null,
+          ${newProject.extras}
+        )
+      """).update.withUniqueGeneratedKeys[Project](
+        "id",
+        "created_at",
+        "modified_at",
+        "created_by",
+        "modified_by",
+        "owner",
+        "name",
+        "slug_label",
+        "description",
+        "visibility",
+        "tile_visibility",
+        "is_aoi_project",
+        "aoi_cadence_millis",
+        "aois_last_checked",
+        "tags",
+        "extent",
+        "manual_order",
+        "is_single_band",
+        "single_band_options",
+        "default_annotation_group",
+        "extras",
+        "default_layer"
       )
-    """).update.withUniqueGeneratedKeys[Project](
-      "id",
-      "created_at",
-      "modified_at",
-      "created_by",
-      "modified_by",
-      "owner",
-      "name",
-      "slug_label",
-      "description",
-      "visibility",
-      "tile_visibility",
-      "is_aoi_project",
-      "aoi_cadence_millis",
-      "aois_last_checked",
-      "tags",
-      "extent",
-      "manual_order",
-      "is_single_band",
-      "single_band_options",
-      "default_annotation_group",
-      "extras"
-    )
+      defaultProjectLayer <- ProjectLayerDao.insertProjectLayer(
+        ProjectLayer(UUID.randomUUID(),
+                     now,
+                     now,
+                     "default_layer",
```

Comment:
This will be a user visible name so we should call it something like "Default scene layer"

Reply:
my point is, no reason for underscores. It's not a variable name.